packages/util/src/util.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
A file named util in util is kind of weird, maybe "formatters.ts"?

Reply:
Nice. renamed to `formatters.ts`